mmaction/models/recognizers/recognizer2d.py

```diff
@@ -11,6 +11,8 @@ class Recognizer2D(BaseRecognizer):
 
     def forward_train(self, imgs, labels, **kwargs):
```

Comment:
Is it right to put `feature_extraction` to `forward_train` rather than `forward_test` ?